src/java.base/share/classes/java/lang/module/Resolver.java

```diff
@@ -572,6 +572,15 @@ private static String toHexString(byte[] ba) {
                         assert requires.modifiers().contains(Modifier.STATIC);
                         continue;
                     }
+
+                    // m2 is automatic module in parent configuration => m1 reads
+                    // all automatic modules that m2 reads.
+                    if (m2.descriptor().isAutomatic()) {
+                        m2.reads()
+                                .stream()
```

Comment:
Nit: not sure if this formatting is intentionally or you meant `m2.reads().stream()` in one line (which I like better).

Reply:
Thanks, I suspect I just hit Return and the IDE formatted it. Fixed now.